### ISSUE TITLE
[Feature Request] Better filter options for routing

### ISSUE THREAD

**Issue by viaj3ro** [Ride-The-Lightning/RTL#632]
due to a very large amount of low value tx, it's becoming increasingly difficult to monitor meaningful traffic on my node. A simple filter in routing would solve the problem:

1. make it possible to filter out tx below a certain value (i.e. 10k sats) or even better, provide an option to filter a certain range.

2. make it possible, to filter out certain nodes via `-nodeID` or `-node alias` (same as google search basically)

**Comment by dscotese** [comment]
The filter function examines short channel IDs, but not channel aliases.  It examines dates, amounts, and fees.  The failure to examine aliases is frustrating.  It ought to filter on whatever it uses to display the the routing.  Should this be a separate issue?

**Comment by viaj3ro** [comment]
channel aliases? Do you mean node aliases? It does filter for node aliases for me and channel aliases I've never heard of.

**Comment by ShahanaFarooqui** [comment]
> The filter function examines short channel IDs, but not channel aliases. It examines dates, amounts, and fees. The failure to examine aliases is frustrating. It ought to filter on whatever it uses to display the the routing. Should this be a separate issue?

It can better be addressed separately. Are you using C-Lightning or Eclair?

**Comment by dscotese** [comment]
> channel aliases? Do you mean node aliases? It does filter for node aliases for me and channel aliases I've never heard of.

Good catch, I totally meant "node aliases".  As you can imagine, I (inadvertently) shortcut the terminology because every channel does have a node on the other side, and therefore an alias.  I will try to be better about it.

> Are you using C-Lightning or Eclair?
I use both, as a matter of fact.  In Eclair, it seems to work just fine.  In CL, I have to go look up the short channel ID even though the routing table displays the alias of the node to which the channel connects.

**Comment by ShahanaFarooqui** [comment]
@dscotese Thanks for the confirmation. Created a new issue #982 to fix the bug for CLT.

**Comment by dscotese** [comment]
@ShahanaFarooqui your title is not accurate.  It filters on short channel ID excellently.  The problem is that it is displaying the node alias instead of the short channel ID (for some channels), and it does NOT filter on the node aliases.

**Comment by viaj3ro** [comment]
I'm confused. I'm on 0.7.0 and I believe it does filter for node aliases for me. Can you specify where exactly it doesn't work for you, so I can double check on my system?

**Comment by ShahanaFarooqui** [comment]
@viaj3ro the issue is with CLightning setup.

**Comment by dscotese** [comment]
> I'm confused. I'm on 0.7.0 and I believe it does filter for node aliases for me. Can you specify where exactly it doesn't work for you, so I can double check on my system?

In the page that displays channels, some are displayed with short channel IDs and some are displayed with Node aliases.
